ui/src/kapacitor/containers/TickscriptPage.js

```diff
@@ -23,6 +23,54 @@ class TickscriptPage extends Component {
       },
       validation: '',
       isEditingID: true,
+      logs: [],
+    }
+  }
+
+  shouldFetch = null
+
+  logKey = j => (log, i) => ({
+    ...log,
+    key: `${log.ts}-${j}-${i}`,
+  })
+
+  fetchChunkedLogs = async (kapacitor, ruleID) => {
+    try {
+      const response = await fetch(
+        `${kapacitor.links.proxy}?path=/kapacitor/v1/logs?task=${ruleID}`,
+        {
+          method: 'GET',
+          headers: {'Content-Type': 'application/json'},
+        }
+      )
+
+      const reader = await response.body.getReader()
+      const decoder = new TextDecoder()
+
+      let result
+      let j = 0
+
+      while (this.shouldFetch === true && !(result && result.done)) {
+        result = await reader.read()
+
+        const chunk = decoder.decode(result.value || new Uint8Array(), {
+          stream: !result.done,
+        })
+
+        const json = `[${chunk.split('}{').join('},{')}]`
```

Comment:
this will be \n rather than }{ soon-ish

Reply:
i guess you could say... that's a new line of reasoning